GLOSSARY.md

```diff
@@ -0,0 +1,44 @@
+# PyTorch Glossary 
+
+#### TorchScript
+An interface to the TorchScript JIT compiler and interpreter.
+
+#### ATen
+Short for "A Tensor Library". The foundational tensor and mathematical
+operation library on which all else is built.
+
+#### Operation
+A unit of work. For example, a MatMul is an operation.
+
+#### Native Operation
+An operation that comes natively with PyTorch ATen.
```

Comment:
An example like above, or maybe an actual Python Operation `torch.add` would be nice

Reply:
Done.